docs/meshstack.metadata-tags.md

```diff
@@ -22,168 +25,42 @@ such metadata include:
 
-### Configuration Model
+> Note: doing the actions below will require the Partner Admin role.
 
-Operators can configure tag schemas in the [meshStack configuration model](meshstack.configuration.md) under `meshfed.web` as follows:
+To get started with managing tags in meshstack, navigate to the Tags page which is available in the Administration area:
 
-```dhall
-{
-  customer-tag-schema = Some "./my-customer-tag-schema.json"
-, project-tag-schema = Some "./my-project-tag-schema.json"
-, landing-zone-tag-schema = Some "./my-landing-zone-tag-schema.json"
-, payment-method-tag-schema = Some "./payment-method-tag-schema.json"
-}
-```
+![Tag List](assets/metadata_tags/admin_area_tags_list.png)
 
-The schemas need to be available in the configuration repository.
+This page will show you all tags that are currently active on supported meshObjects. If you would like to know more details about a tag, or create, edit, or delete a tag, you can do so by clicking on the large arrow ▶️ on the right for a meshObject.
+This next page will show you more details about a tag such as the title, description or data type of a tag.
 
-### Tags on Payment Methods
+### Defining a new tag
 
-[Payment Methods](meshcloud.payment-methods.md) can also provide tag information. For example,
-the default "cost center" payment method provides the `costCenter` tag.
+You can define a new tag by clicking '+ Create' at the top right in the page mentioned above.
 
-Operators can also add additional tags to [Payment Methods](meshcloud.payment-methods.md).
-This is useful if your organization needs additional metadata like e.g. contract or budget numbers to chargeback cloud costs.
+You will be prompted with a new page where you can enter a lot of information. We will briefly explain what each fields means and why you should use it.
+
+> Hint: after entering a name, you can view the tag input as if it were shown to the end user. You can do this on the right-side of the screen where it says 'Tag Preview'.
+> This can help you get an idea of how it will look when end-users fill in the tag value.
+
+- **Display Name**: This is the text that will be shown above the input field which the user reads. Note that is **not** the name of the tag itself.
+- **Name**: This is the actual name of the tag as it will be displayed in the green bubble. It is also the name that is used for uniquely identifying the tag and when exporting it or when [using it within a landing zone](#exposing-tags-via-landing-zones). This name cannot be changed later (at the moment). We recommend using a name that is similar or the same as the Display Name. Keep in mind that the use of spaces is not allowed since the tag name is used for technical purposes.
+- **Description (optional)**: This is the description of the tag that will be displayed in small grey text above the tag input. It is useful for giving extra information to the user when the (display) name of the tag is not explanatory enough.
```

Comment:
although i get what you want to say, I find the `(display)` a bit strange. I would probably just omit it.